ui_graphical.ks.in

```diff
@@ -0,0 +1,15 @@
+#version=DEVEL
+#test name: ui_graphical
+
+# Use defaults.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+# Run the installation in the graphical mode.
+graphical
+
+%post
+# The installation is successful.
+%ksappend validation/success_if_result_empty.ks
```

Comment:
I think there should be an additional check that the installation was really executed in graphical/text/cmdline mode - for example when there is a problem with X, anaconda fallbacks to text mode:

Starting installer, one moment...
anaconda 33.16.3.26-1.el8 for Red Hat Enterprise Linux 8.3 started.
 * installation log files are stored in /tmp during the installation
 * shell is available on TTY2
 * when reporting a bug add logs from /tmp as separate text/plain attachments
13:16:46 X startup failed, falling back to text mode
================================================================================
================================================================================

Reply:
Fixed.